### ISSUE TITLE
Stuck to get good result with i2v wan 2.2

### ISSUE THREAD

**Issue by BenDes21** [kohya-ss/musubi-tuner#579]
Hi there, Im trying since few days to train my dataset of videos for a motion lora but even at 20 epochs which is the end of my training,  I don't have anything related to the original motion of my videos ( even a beginning ) .   My dataset is currently 10 videos of 10s / 480p / 24fps

Here the dataset file :  

[[datasets]]
video_directory = "/workspace/motion_POV"
cache_directory = "/workspace/motion_POV"
caption_extension = ".txt"
enable_bucket = true

resolution = [854, 480]
max_frames = 129
frame_extraction = "full"

And the High config  ( low is same except the path model ) :  

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 src/musubi_tuner/wan_train_network.py \
    --max_train_epochs 20 --save_every_n_epochs 1 --seed 42 \
    --output_dir /workspace/output --output_name  motion_HIGH \
    --network_module networks.lora_wan \
    --task i2v-A14B \
    --dataset_config /workspace/dataset.toml \
    --min_timestep 900 --max_timestep 1000 \
    --dit /workspace/models/wan2.2_i2v_high_noise_14B_fp16.safetensors \
    --t5 /workspace/models/umt5-xxl-enc-bf16.safetensors \
    --vae /workspace/models/wan_2.1_vae.safetensors \
    --discrete_flow_shift 5.0 --timestep_sampling shift  \
    --rope_func comfy --sdpa --preserve_distribution_shape --gradient_checkpointing  \
    --fp8_base --fp8_scaled --mixed_precision fp16 \
    --max_data_loader_n_workers 32 --persistent_data_loader_workers \
    --network_module networks.lora_wan --network_dim 16 --network_alpha 16 --network_args loraplus_lr_ratio=4 \
    --optimizer_type schedulefree.RAdamScheduleFree --learning_rate 3e-5 

Is anything wrong with my dataset / config file ? Feel like to I need to push more epochs ( probably 50 ) or add a repeat of 2-3 for my dataset. Not super familiar with RAdamScheduleFree also
